src/app/components/pagination-component/pagination.component.scss

```diff
@@ -39,17 +39,19 @@ ngb-pagination {
     width: 16px;
     height: 16px;
     content: '';
-    background-color: currentColor;
+    background-color: currentcolor;
```

Comment:
Please check, no such property exists. [https://saje.hashnode.dev/css-currentcolor](url)

Reply:
Please check other documentation
https://developer.mozilla.org/en-US/docs/Web/CSS/color#values

Reply:
They both valid, but In this case camel case violates the rule of the linter for CSS properties and I see no point in adding an exception if it can be fixed